src/middleware/assets.js

```diff
@@ -77,6 +80,20 @@ Or \`rm -rf bower_components/n-ui && bower install n-ui\` if you're no longer wo
 			return currentHeadCsses;
 		}, {}) : {};
 
+	if (process.argv.indexOf('--dev') !== 0) {
```

Comment:
Not sure what is most robust, but process.NODE_ENV !== 'production' is another option.